nodepool/config/templates/cluster.yaml

```diff
@@ -46,6 +46,11 @@ availabilityZone: {{.AvailabilityZone}}
 # ARN of the KMS key used to encrypt TLS assets.
 kmsKeyArn: "{{.KMSKeyARN}}"
 
+# If you do not want kube-aws to manage certificaes, set it to false. If you do that
+# you are responsible for making sure that nodes have correct certificates by the time
+# daemons start up.
```

Comment:
An example (or a link to a doc explaining it) of how to feed certificates to nodes before daemons start up would be helpful. I'm not really sure how myself and so are others!

Also, a brief explanation of when this feature is intended to be used (a.k.a why "you do not want kube-aws to manage certificates" in the first place) would be good to add, too.